public/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Steam</title>
+</head>
+<body style="background: #1b2837; font-family: Arial, Helvetica, sans-serif; color: #c6d4df;">
+  <div id="app"></div>
+  <div id="metagame"></div>
+  <script src="http://localhost:3005/bundle.js"></script>
```

Comment:
Similarly, I would change the name of your bundle file to something more particular to your service, such as "metadataBundle.js."